packages/cardano-services/test/Metadata/HttpMetadataService.test.ts

```diff
@@ -0,0 +1,172 @@
+/* eslint-disable max-len */
+import { Cardano, ProviderError, ProviderFailure } from '@cardano-sdk/core';
+import { ExtMetadataFormat, createHttpMetadataService } from '../../src/Metadata';
+import { adaPoolsExtMetadataMock, cip6ExtMetadataMock, mainExtMetadataMock } from './mocks';
+import { createGenericMockServer } from '@cardano-sdk/util-dev';
+import { dummyLogger } from 'ts-log';
```

Comment:
We should probably always be using the test logger now

Reply:
As I see we still use `dummyLogger` in a lot of places across service unit tests, ex. [here](https://github.com/input-output-hk/cardano-js-sdk/blob/159535092033a745664c399ee1273da436fd3374/packages/cardano-services/test/NetworkInfo/NetworkInfoHttpService.test.ts#L19), should I update the usage of test logger across all tests as a separate commit?

Reply:
Agree

Reply:
d267744a4b03ced7a9f0769bc532c3a462b94575